src/main/app/src/assets/i18n/en.json

```
@@ -280,6 +282,9 @@
   "zaakOpschorten": "Suspend the case",
   "actief": "Active",
   "inactief": "Inactive",
+  "besluit.publicatie.indicatie.koptitel": "Publication",
+  "besluit.publicatie.indicatie.onderschrift": " Publication term: {{publicationTermDays}} day",
```

```suggestion
  "besluit.publicatie.indicatie.onderschrift": "Publication term: {{publicationTermDays}} day",
```

? Leading space seems wrong?